common/client/client.go

```diff
@@ -0,0 +1,27 @@
+package client
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/twitter/scoot/common/dialer"
+	"github.com/twitter/scoot/scheduler/client"
+)
+
+// Client interface that includes CLI handling
+type CLIClient interface {
+	Exec() error
+}
+
+// Implements CLIClient - basic
```

Comment:
Does this actually implement CLIClient iface? I don't see the Exec() implementation - looks like we use this as base struct ontop of which we build actual implementations - can the comment make that clearer?